file_format.md

```diff
@@ -22,9 +33,13 @@ JSON files accepted by the simularium-viewer contain the following data in JSON
       * ex: "actin#barbed_ATP_1" is parsed as agent type "actin" in states "barbed", "ATP", and "1"
       * ex: "actA" is parsed as agent type "actA" with no state information
       * if no name is provided, the agent type ID, an integer number, is used for display
-    * pdb (optional) - the filename of the PDB file to render for this agent. If this field isn’t provided or if the file isn’t found, the renderer will fall back to mesh rendering
-    * mesh (optional) - the filename of the OBJ mesh file to render for this agent. If this field isn’t provided or if the file isn’t found, the renderer will fall back to a sphere
-    * PDB and mesh data is currently only used for streaming trajectories, but this will be updated soon
+    * geometry - rendering information for each agent type (note: only the first 100,000 geometry files will be loaded)
+      * displayType - “SPHERE”, “CUBE”, “GIZMO”, “FIBER”, “PDB”, or “OBJ”
```

Comment:
I believe CUBE, GIZMO, and FIBER are unsupported.  I'd remove em.  FIBER is a very confusing one, because we still have this per-agent visType which is basically fiber or not fiber.  I am really not sure if the displayable type should be per-agent or per-agenttype. If we project into the future of multiscale where fibers turn into collections of agents and back, that might shed light on what to do.  gut feeling at the moment is that the per-agent vistype should go away.

Reply:
removed the others, but FIBER is still a display type and a viz type afaik. Does the viewer only use the viz type? I also think it should be removed per agent and we just use the display type and subpoints, but that's a larger discussion

Reply:
fiber is not a recognized display type in the viewer yet.   currently the viewer uses the viz type.  We have more in-depth discussion about this in other issues/PRs.